docs/pages/versions/unversioned/guides/configuring-statusbar.md

```diff
@@ -13,35 +13,57 @@ The configuration for Android status bar lives under the `androidStatusBar` key
 This option can be used to specify whether the status bar content (icons and text in the status bar) is light, or dark. Usually a status bar with a light background has dark content, and a status bar with a dark background has light content.
 
 The valid values are:
-
 - `light-content` - The status bar content is light colored (usually white). This is the default value.
 - `dark-content` - The status bar content is dark colored (usually dark grey). This is only available on Android 6.0 onwards. It will fallback to `light-content` in older versions.
 
+> Note: Be aware that choosing `light-content` and having `SplashScreen` image that is very light (close to `white` color) and/or having `backgroundColor` of your app set to `white` (or close to `white` color) may result in status bar's icons blending in and wouldn't be visible.
+
 ### `backgroundColor`
 
 This option can be used to set a background color for the status bar.
 
-Keep in mind that the Android status bar is translucent by default in Expo apps. But, when you specify an opaque background color for the status bar, it'll lose it's translucency.
+Keep in mind that the Android status bar can be `translucent` (default behavior for Expo apps).
```

Comment:
```suggestion
This defaults to  `translucent` in Expo apps.
```

Reply:
I've removed this one as it stopped to make any sense, but I've described a little bit more what might happen in `### translucent` section - please take a look at this 😉